### PR TITLE
add mva Id for electrons

### PR DESCRIPTION
**Pull request by alesaggio**
1) The Iso variable specified at https://github.com/latinos/LatinoAnalysis/blob/de026c531cec33d6f1cb999ec720e52692596116/Gardener/python/variables/LeptonSel_cfg.py is 'relPFIsoRhoCorr'. The one that I used (from the FW) is 'relativeIsoR04_withEA', although I didn't find any info whether the 'relPFIsoRhoCorr' implies a cone of 0.4 or 0.3 (in the latter case I should have used 'relativeIsoR03_withEA'). Any idea on how to get this info?

2) As discussed yesterday with @OlivierBondu, I'm selecting electrons with 'mvaEleID-Spring16-GeneralPurpose-V1-wp90'. Any comment/remark?

3) Should I completely remove the ICHEP'16 IDs?

### CODE REVIEW

File: plugins/HtoZAAnalyzer.cc [hunk 1]
```diff
@@ -328,7 +330,7 @@ void HtoZAAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, con
         {
             // some selection
             // Ask for medium ID
```
Comment (OlivierBondu) [inline plugins/HtoZAAnalyzer.cc]: is now loose :)

File: test/HtoZAConfiguration.py [hunk 2]
```diff
@@ -121,19 +123,17 @@
...
             # Signal: H->ZA - mH=500, mA=100
             '/store/mc/RunIISpring16MiniAODv2/HToZATo2L2B_MH-500_MA-100_13TeV-madgraph-pythia8/MINIAODSIM/PUSpring16RAWAODSIM_reHLT_80X_mcRun2_asymptotic_v14-v1/20000/1287F415-023B-E611-A374-FA163E48F4D0.root'
 
-            # TT
+            # Background: TT
             # '/store/mc/RunIISummer16MiniAODv2/TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8/MINIAODSIM/PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/120000/00ED79D3-CFC1-E611-B748-3417EBE64483.root'
 
-            # DY
+            # Background: DY
             # '/store/mc/RunIISummer16MiniAODv2/DYToLL_2J_13TeV-amcatnloFXFX-pythia8/MINIAODSIM/PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v2/100000/00CEFB4F-C1D2-E611-BBF4-7845C4FC3C11.root<Paste>'
```
Comment (OlivierBondu) [inline test/HtoZAConfiguration.py:137]: could you also please update the test files to more recent stuff ?

File: plugins/HtoZAAnalyzer.cc [hunk 3]
```diff
@@ -307,13 +307,15 @@ void HtoZAAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, con
         
         bool result = allelectrons.ids[index][m_electron_hlt_safe_wp_name];
 
-        // Add dxy and dz cuts described at https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedElectronIdentificationRun2#Offline_selection_criteria
+        // Add dxy and dz cuts described at https://github.com/latinos/LatinoAnalysis/blob/master/Gardener/python/variables/LeptonSel_cfg.py
```
Comment (OlivierBondu) [inline plugins/HtoZAAnalyzer.cc]: please also have the git hash in the url for bookkeeping :)

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
thanks ! only minor changes :)

**Comment by alesaggio** [comment]
PR updated ;)

**Review by OlivierBondu — APPROVED** [review]

**Comment by OlivierBondu** [comment]
Thanks !

**Pull request merged**